src/components/Gauge.js

```diff
@@ -76,119 +53,124 @@ const StatusGaugeWrapper = styled.div`
   }
 `
 
-const GaugeTotalText = styled.text`
-  font-family: ${font};
-  font-size: ${props => px(props.count > 999 ? 20 : 26)};
-  fill: #5e5e5e;
-  letter-spacing: -1px;
-  text-anchor: middle;
-  transform: translateX(0);
-`
+class StatusGauge extends React.Component {
+  constructor (props) {
+    super(props)
+
+    const { data } = props
+
+    const values = data.map(x => x.value)
+    const labels = data.map(x => x.name)
+    const colors = data.map(x => x.color)
+    const count = values.reduce((carry, item) => carry + item, 0)
```

Comment:
This means that the component won't update if you change the props after it mounts